src/Event/ReadModel/RDF/RdfProjector.php

```diff
@@ -251,6 +255,27 @@ private function handleTypeUpdated(TypeUpdated $event, string $uri, Graph $graph
         $this->graphRepository->save($uri, $graph);
     }
 
+    private function deleteLocation(string $uri, Graph $graph): void
+    {
+        $resource = $graph->resource($uri);
+
+        $resource->set(self::PROPERTY_ACTVITEIT_LOCATIE, null);
+    }
+
+    private function addLocation(string $uri, Graph $graph): void
+    {
+        $locationId = $this->locationIdRepository->get($uri);
+
+        if ($locationId === null) {
```

Comment:
in `addLocationOnSpaceTimeResource()` there is no check on whether the `locationIdRepository` returns `null`.  Is this correct?

Reply:
That is indeed not needed because `CalendarUpdated` happens after `LocationUpdated`.

But while thinking about this question I noticed that a scenario is missing. When an extra `LocationUpdated` would happen after the initial `CalendarUpdated` the location would be added to the root of the RDF projection because it does not take into account the calendar type.

I will add extra unit tests for this.